WORKSPACE

```diff
@@ -46,22 +46,15 @@ ENVOY_REPOSITORY = "https://github.com/envoyproxy/envoy-wasm"
 
 ENVOY_PREFIX = "envoy-wasm-"
 
-LOCAL_ENVOY_PROJECT = "/PATH/TO/ENVOY"
-
 envoy_repository_rule(
     name = "envoy",
     prefix = ENVOY_PREFIX,
     repository = ENVOY_REPOSITORY,
     sha = ENVOY_SHA,
     sha256 = ENVOY_SHA256,
 )
-
-# TODO(silentdai) Use bazel args to select envoy between local or http
-# Uncomment below and comment above http_archive to depends on local envoy.
-#local_repository(
-#     name = "envoy",
-#     path = LOCAL_ENVOY_PROJECT,
-#)
+# To override with local envoy, just pass `--override_repository=envoy=/PATH/TO/ENVOY` or
+# persist the option in `user.bazelrc`.
```

Comment:
Nit: prefer this before the `envoy_repository_rule`.

Reply:
ok will let #2505 merge first and fix this and conflict.